### ISSUE TITLE
Training does not show

### ISSUE THREAD

**Issue by Greenlili** [znxlwm/pytorch-MNIST-CelebA-GAN-DCGAN#13]
Hello, your code is very good, but I have some problems that I can't solve.When I replace

transform = transforms.Compose([
        transforms.Scale(img_size),
        transforms.ToTensor(),
        transforms.Normalize(mean=[0.5], std=[0.5])
])
and
D_train_loss.item()

And then you get these two warnings

UserWarning: volatile was removed and now has no effect. Use `with torch.no_grad():` instead.
  fixed_z_ = Variable(fixed_z_.cuda(),volatile=True)
UserWarning: The use of the transforms.Scale transform is deprecated, please use transforms.Resize instead.
  "please use transforms.Resize instead.")

and after that the program keeps running, but it doesn't do any calculations like output losses and so forth.I would be most grateful if you could answer me
![image](https://user-images.githubusercontent.com/85828983/143450564-8843fdd3-4600-49d3-aa19-bb0155367607.png)
